pyterrier/datasets.py

```diff
@@ -718,6 +743,9 @@ def _merge_years(self, component, variant):
     "info_url" : "https://trec.nist.gov/data/precmed.html",
 }
 
+def trec2text(dataset):
```

Comment:
delete this for now and make a new issue for it?

Reply:
yes, woops